kvmagent/kvmagent/test/vm_plugin_testsuite/test_vm_dump.py

```
@@ -0,0 +1,38 @@
+__ENV_SETUP__ = {
+    'self': {
+    }
+}
+
+
+class TestVmPlugin(TestCase, vm_utils.VmPluginTestStub):
+    @classmethod
+    def setUpClass(cls):
+        network_utils.create_default_bridge_if_not_exist()
+
+    @env.test_for(handlers=[
```

Ruff (0.11.9)

`env` may be undefined, or defined from star imports

(F405)